docs/Configuring-Pantheon/Testing-Developing-Nodes.md

```diff
@@ -21,18 +21,22 @@ To start a bootnode for a private network:
 
     !!! example
         ```bash
-        pantheon export-pub-key bootnode
+        pantheon --genesis=privateNetworkGenesis.json --datadir=nodeDataDir export-pub-key bootnode
```

Comment:
If we intend the reader to change the directory to suit their own purpose, I don't see much difference between a one-word placeholder vs a dummy path. OTOH, at least you could use the temp path if you don't care where the data is stored.

Reply:
@jakehaugen - see above.  I'm not sure about using tmp because if they set up a private network following these instructions and then want to use it for testing, they don't really want to use a tmp directory.  Have you got any thoughts on best practices for specifying directories in examples like this?